### ISSUE TITLE
The beginning parts of the path are cut off when executing these commands

### ISSUE THREAD

**Issue by JWaldenback** [knausj85/knausj_talon#1317]
https://github.com/talonhub/community/blob/72111ebc38e1498e8a489a517bfe24bd42bcb744/apps/windows_explorer/windows_explorer.py#L110

https://github.com/talonhub/community/blob/72111ebc38e1498e8a489a517bfe24bd42bcb744/apps/windows_explorer/windows_explorer.py#L119

Adding a short sleep time solves the issue 🙂

**Comment by AndreasArvidsson** [comment]
Yeah having to add sleeps is unfortunately the reality when we are relying on key presses. The ui is not immediately updated and we start typing before it's ready. If you have figured out their minimum sleep we need please make a pull request with those changes :)
